.travis.yml

```diff
@@ -7,3 +7,7 @@ cache: cargo
 matrix:
   allow_failures:
     - rust: nightly
+
+script:
+  - cargo test --verbose
+  - cargo test --verbose --no-default-features
```

Comment:
I guess `cargo test --no-default-features` will (at least in this case) run strictly fewer tests. However, it'll be useful to check that the tests at least still compile when serialization is disabled, so I'd use:
```yaml
cargo test --verbose --no-default-features --no-run
```

Reply:
Good idea.